vhdbuilder/scripts/gc.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -euxo pipefail
+
+[ -z "${SUBSCRIPTION_ID:-}" ] && echo "SUBSCRIPTION_ID must be set" && exit 1
+
+SKIP_TAG_NAME="${SKIP_TAG_NAME:-gc.skip}"
+SKIP_TAG_VALUE="${SKIP_TAG_VALUE:-true}"
```

Comment:
I would probably just embed it in the script.
It will be really hard to find out If someone going to change it in a pipeline.